class-lab/exercise4/TextboxWithCounter.js

```diff
@@ -0,0 +1,23 @@
+class TextboxWithCounter {
+  constructor(el) {
+    el.innerHTML = `
+    <p class="counter">0</p>
+    <textarea></textarea>`
+    this.counter = el.querySelector('.counter');
+    this.textarea = el.querySelector('textarea');
+    this.textarea.addEventListener('keyup',
```

Comment:
I'd use the `input` event instead of `keyup` as `input` also catches paste

Reply:
Thanks for the input 'input' :-) - I have replaced it (actually it is only to catch mouse pate)